velox/functions/prestosql/window/tests/WindowTestBase.h

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "velox/exec/tests/utils/OperatorTestBase.h"
+#include "velox/exec/tests/utils/PlanBuilder.h"
+#include "velox/functions/prestosql/window/WindowFunctionsRegistration.h"
+#include "velox/vector/fuzzer/VectorFuzzer.h"
+
+using namespace facebook::velox::exec::test;
+
+namespace facebook::velox::window::test {
+
+class WindowTestBase : public OperatorTestBase {
+ protected:
+  void SetUp() override {
+    exec::test::OperatorTestBase::SetUp();
+    velox::window::registerWindowFunctions();
+  }
+
+  std::vector<RowVectorPtr>
+  makeVectors(const RowTypePtr& rowType, vector_size_t size, int numVectors) {
```

Comment:
I missed this, but would you move the implementation to the .cpp file to help speed up builds?

Reply:
Done.